analyzers/tests/SonarAnalyzer.Test/TestCases/AspNet/AvoidUnderPosting.CSharp9.cs

```diff
@@ -6,8 +6,9 @@ namespace CSharp9
 {
     // https://learn.microsoft.com/en-us/aspnet/core/mvc/models/model-binding#constructor-binding-and-record-types
     public record RecordModel(
-        int ValueProperty,                                                          // Noncompliant
-        [property: JsonRequired] int RequiredValueProperty,                         // without the property prefix the attribute would have been applied to the constructor parameter instead
+        int ValueProperty,                                                         // Noncompliant
+        [property: JsonRequired] int RequiredValueProperty,                        // Without the property prefix the attribute would have been applied to the constructor parameter instead
+        [Range(0, 2)] int RequiredValuePropertyWithoutPropertyPrefix,              // Noncompliant, FP see: https://github.com/SonarSource/sonar-dotnet/issues/9363
```

Comment:
```suggestion
        [property: Range(0, 2)] int RequiredValuePropertyWithoutPropertyPrefix,              // Noncompliant, FP see: https://github.com/SonarSource/sonar-dotnet/issues/9363
```

Reply:
As discussed I'll add a clarifying comment and leave as is.
The point here is that the attribute is applied on the parameter and it's still serves its purpose, but we don't detect it.